README.md

```diff
@@ -4,6 +4,8 @@
 
 To connect to the databases successfully you will need to run npm i and set up a .env.development and .env.test file, which include PGDATABASE=<DATABASE_NAME_HERE>
 
+To create the databases, run $ npm setup-dbs
```

Comment:
Take another look at your Trello board and decide what you need to do with your README.